db/schema.rb

```diff
@@ -77,7 +77,7 @@
   create_table "relationships", force: :cascade do |t|
     t.bigint "padawan_id", null: false
     t.bigint "mentor_id", null: false
-    t.integer "status", default: 0
```

Comment:
Need to `rails db:drop db:create db:migrate db:seed` to keep Alex's modification